### DISCUSSION TITLE
Lambda function code size

### DISCUSSION THREAD

**Discussion by sebastijan94** [SamWSoftware/serverless-auto-swagger#81] [Q&A]
Using serverless-auto-swagger will increase the size of the lambda function to more than 40 MB. Is it possible to somehow reduce this size?

**Comment by bfaulk96** [comment]
Thanks for bringing this up. I just tested on my own machine and see similar (see the unzipped numbers):

with:
- `.serverless` folder: **33.8MB**
  - `cloudformation-template-update.json`
  - `serverless-state.json`
  - `my-project.zip`
- `my-project` folder, unzipped: **185.9MB**

without:
- `.serverless` folder: **33.8MB**
  - `cloudformation-template-update.json`
  - `serverless-state.json`
  - `my-project.zip`
- `my-project` folder, unzipped: **152.9MB**

I'll try to look into this at some point, although honestly I don't know when that will be. I'm pretty busy for the next month at least.
